src/app/components/core/south/south-plugin-modal/south-plugin-modal.component.ts

```diff
@@ -0,0 +1,114 @@
+import { Component, OnInit, Input, OnChanges, ViewChild, ElementRef, Output, EventEmitter } from '@angular/core';
```

Comment:
let's keep a common PluginModalComponent with 2 class level variables this.plugins and this.pluginType (plugin type should be set as per the wizard (step) itself. north, notifications rule/notify delivery, filters ...